Assets/SEE/Utils/VersionControlInterface.cs

```diff
@@ -0,0 +1,29 @@
+/// <summary>
```

Comment:
See my above note on `SwitchVersionControlSystems`: This must be nested in a namespace and it should rather be somewhere else in a new version-control layer, not part of `SEE.Utils`.

Reply:
It's fixed.